_inc/client/lib/analytics/index.js

```diff
@@ -102,22 +108,30 @@ const analytics = {
 
 	tracks: {
 		recordEvent: function( eventName, eventProperties ) {
-			let superProperties;
+			let blogIdProperty,
+				superProperties;
 
 			eventProperties = eventProperties || {};
 
-			debug( 'Record event "%s" called with props %s', eventName, JSON.stringify( eventProperties ) );
 			if ( eventName.indexOf( 'akismet_' ) !== 0 && eventName.indexOf( 'jetpack_' ) !== 0 ) {
 				debug( '- Event name must be prefixed by "akismet_" or "jetpack_"' );
 				return;
 			}
 
+			if ( _blog ) {
```

Comment:
This seems to be doing the same we're doing for `superProps`. Do you think we could set the blog ID as a superProp? This way we won't have to alter the `analytics.initialize` method signature, and will use what we already have.

Reply:
yes, I agree.

Reply:
I'd prefer to keep it at the same level as `userId` for the sake of future analyses (and data logic).

Reply:
I'm not fully convinced this would be convenient on the data analysis end, but since here it makes sense and I'm roughly guessing, taking up on your suggestion :)